### PR TITLE
feat(pci): display region with quota overflow disabled

### PR DESCRIPTION
**Pull request by marie-j**
MANAGER-3007

### CODE REVIEW

File: packages/manager/modules/pci/src/components/project/regions-list/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "pci_project_regions_list_continent_all": "Toutes les localisations",
   "pci_project_regions_list_empty": "Aucune localisation n'est disponible",
-  "pci_project_regions_list_add": "Nous vous invitons à activer les dernières localisations de chaque zone dans la partie <a href=\"{{url}}\" class=\"oui-link\">Localisations</a>"
+  "pci_project_regions_list_add": "Nous vous invitons à activer les dernières localisations de chaque zone dans la partie <a href=\"{{url}}\" class=\"oui-link\">Localisations</a>",
+  "pci_project_regions_list_region": "Région",
```
Comment (jleveugle) [inline packages/manager/modules/pci/src/components/project/regions-list/translations/Messages_fr_FR.json]: `Localisation` vs `Region` ? Should we use the same word ? :)

**Comment by ovh-cds** [comment]
CDS Report pci#3646.0 ✘  
* Stage 1
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

**Comment by ovh-cds** [comment]
CDS Report freefax#3646.0 ✘  
* Stage 1
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

**Comment by ovh-cds** [comment]
CDS Report telecom-dashboard#3646.0 ✘  
* Stage 1
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js [hunk 2]
```diff
@@ -75,6 +72,16 @@ export default class RegionsListController {
     }
   }
 
+  static isRegionDisabled(region) {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js]:
```suggestion
  static isRegionDisabled([region]) {
      return !get(region, 'hasEnoughQuota', false);
  }
```
but I don't think this is necessarily better

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js [hunk 3]
```diff
@@ -75,6 +72,16 @@ export default class RegionsListController {
...
+    return region.length === 1 && !region[0].hasEnoughQuota;
+  }
+
+  onMacroChange(macro, regions) {
+    if (regions.length === 1) {
+      this.onRegionChange(regions[0]);
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js]:
```suggestion
      this.onRegionChange(first(regions));
```
not necessarily better too

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.html [hunk 4]
```diff
@@ -26,6 +12,50 @@
                         }">
                     </span>
                 </p>
+
+                <div class="col-md-6 col-xl-4 my-3" data-ng-repeat="(macro, regions) in $ctrl.regionsByContinents[continent]">
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/regions-list/regions-list.html]:
```suggestion
                <div class="col-md-6 col-xl-4 my-3" data-ng-repeat="(macro, regions) in ::$ctrl.regionsByContinents[continent] track by macro">
```
?

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.html [hunk 5]
```diff
@@ -26,6 +12,50 @@
...
+                    <oui-select-picker
+                        data-name="{{continent}}_regions"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/regions-list/regions-list.html]:
```suggestion
                        data-name="{{:: continent}}_regions"
```

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.html [hunk 6]
```diff
@@ -26,6 +12,50 @@
...
+                        data-model="$ctrl.macroRegion"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/regions-list/regions-list.html]:
```suggestion
                        data-model=":: $ctrl.macroRegion"
```
and so on

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.html [hunk 7]
```diff
@@ -26,6 +12,50 @@
...
+                        data-label="{{ regions.length === 1 ? regions[0].microRegion.text : macro }}"
+                        data-values="[macro]"
+                        data-variant="light"
+                        data-on-change="$ctrl.onMacroChange(modelValue, regions)"
+                        data-disabled="$ctrl.isRegionDisabled(regions)"
+                        data-required>
+                        <oui-select-picker-section data-ng-if="$ctrl.isRegionDisabled(regions)">
+                            <a class="oui-button oui-button_link"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/regions-list/regions-list.html]:
```suggestion
                            <a class="oui-button oui-button_link"
                                data-translate="pci_project_regions_list_quota"></a>
```

File: packages/manager/modules/pci/src/components/project/regions-list/regions-list.html (hunk 6 above)
Comment (marie-j) [reply packages/manager/modules/pci/src/components/project/regions-list/regions-list.html]: I'm not sure we want `model` to be one-time binded

Comment (FredericEspiau) [reply packages/manager/modules/pci/src/components/project/regions-list/regions-list.html]: Indeed

**Comment by ovh-cds** [comment]
CDS Report overthebox#3788.0 ✘  
* Stage 1
  * Build CA ✔ 
  * Build EU ✔ 
  * Build US ✘

**Pull request merged**